frontend/src/metabase/components/EntityMenu/EntityMenu.jsx

```diff
@@ -52,148 +45,97 @@ class EntityMenu extends Component {
   };
 
   render() {
-    const preferReducedMotion = isReducedMotionPreferred();
-
     const {
       items,
       triggerIcon,
       triggerProps,
       className,
       openClassNames,
       closedClassNames,
-      horizontalAttachments,
       minWidth,
       tooltip,
       trigger,
       renderTrigger,
-      targetOffsetY,
       triggerAriaLabel,
       tooltipPlacement,
     } = this.props;
     const { open, menuItemContent } = this.state;
     return (
-      <Container
+      <Popover
         className={cx(className, open ? openClassNames : closedClassNames)}
-        open={open}
```

Comment:
this _seems_ to work correctly in an uncontrolled state, merely tracking the `open` state for the sake of other visual changes (like showing a different icon when open)

Reply:
this was false. I made it controlled. 😁